docs/best-practice/limit-and-offset-vs-cursor-api-and-hasmore.md

```diff
@@ -0,0 +1,166 @@
+---
+sidebar_position: 50
```

Comment:
Why are there multiple sidebar_positions at 50? Please put them in numerical order, preferably using two-digit numbers (10, 20, 30). This makes it easier to deal add files in later.